README.md

```diff
@@ -97,6 +98,9 @@ The plugin has several configuration fields, this section explains each field us
 }
 ```
 
+`periodicUpdateInterval` is the time interval in seconds to update the resouces accourding to host devices changes if there changed.
+Note: if `periodicUpdateInterval` is 0 then periodic update for host devices will be disabled.
```

Comment:
This breaks current default behaviour.

Reply:
If 0 then sue current default value?

Reply:
how will you disable it ?

Reply:
then go routine of periodic will not run, since the condition of the periodic interval is not positive, check here:
https://github.com/Mellanox/k8s-rdma-shared-dev-plugin/blob/master/pkg/resources/resources_manager.go#L288

Reply:
updated